antidoteclient_test.go

```diff
@@ -385,3 +385,56 @@ func testReadMany(t *testing.T) {
 
 	fmt.Print(counterVal)
 }
+
+func TestMapListMapEntries(t *testing.T) {
+	client, err := NewClient(Host{"127.0.0.1", 8087})
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer client.Close()
+
+	tx, err := client.StartTransaction()
+	if err != nil {
+		t.Fatal(err)
+	}
+	timestamp := time.Now().Unix()
+	bucketname := fmt.Sprintf("bucket%d", timestamp)
+	bucket := Bucket{[]byte(bucketname)}
+	key := Key("keyMap")
+
+	err = bucket.Update(tx, MapUpdate(key,
+		CounterInc(Key("counter"), 13),
+		RegPut(Key("reg"), []byte("Hello World")),
+		SetAdd(Key("set"), []byte("A"), []byte("B"))))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	mapV, err := bucket.ReadMap(tx, key)
+	if err != nil {
+		t.Fatal(err)
+	}
+	entryList, err := mapV.ListMapEntries()
+	if err != nil {
+		t.Fatal(err)
+	}
+	for _, expected := range []struct {
+		key      []byte
+		crdtType CRDTType
+	}{
```

Comment:
I think this could be simplified to use the `MapEntry` type:
```
for _, expected := range []MapEntry{
```

Reply:
Addressed